src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -33,7 +40,22 @@ const MarginWrapper = styled.div`
   }
 `;
 
-const CanonicalMostRead = ({ endpoint, maxTwoColumns }) => {
+const MaxWidthWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_4_SCREEN_WIDTH_MAX}) {
+    margin: 0 ${GEL_MARGIN_ABOVE_400PX} ${GEL_SPACING_QUAD};
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin-bottom: ${GEL_SPACING_QUIN};
+  }
+  @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto ${GEL_SPACING_TRPL};
+    max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
+  }
+
+  margin: 0 ${GEL_MARGIN_BELOW_400PX} ${GEL_SPACING_TRPL};
```

Comment:
Can we place this style before the `media queries`?

Reply:
Updated and moved this into a media query section. Also cleaned up a little overlapping